DFE-Digital/technical-guidance
# 760 technical guidance migration

## Description
Deploy the technical guidance as static html container on AKS
This PR allows deploying using terraform via make commands or with the pipeline
The custom domains will be added later on

## How to review
- Check the review app as linked below (on test.teacherservices.cloud)
- Check the production one: https://technical-guidance-production.teacherservices.cloud/
